app/views/students/_left_nav.html.haml

```diff
@@ -0,0 +1,9 @@
+%div{ class: 'small-2 columns left_nav' }
+  %ul{ class: 'vertical menu'}
```

Comment:
Hash attribute should end with one space before the closing brace<br>Avoid defining `class` in attributes hash for static class names